src/main/java/com/example/springbootlab/controller/CategoryController.java

```diff
@@ -0,0 +1,60 @@
+package com.example.springbootlab.controller;
+
+
+import com.example.springbootlab.entity.Category;
+import com.example.springbootlab.repository.CategoryRepository;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("category")

```

Comment:
It's supposed to be a / before category, and may I suggest you change it to plural, categories.